docker/README.md

```diff
@@ -0,0 +1,64 @@
+# Using ChainerMN via Docker
+
+This directory contains a `Dockerfile` to quickly get up and running with
+ChainerMN via [Docker](http://www.docker.com/).
```

Comment:
Could you add a note that this dockerfile does not support fast interconnect? like:

  If you run ChainerMN on mutiple computing nodes, we strongly 
  recommend to use high speed interconnect such as Infiniband 
  or OmniPath to obtain maximum performance of ChainerMN.

because to obtain the best performance shown in Takuya in GTC'17, fast interconnect is a must.